run_create_eb_windows.py

```diff
@@ -31,6 +31,10 @@ def run_create_eb_windows(name, settings):
     service_name = env['common'].get('SERVICE_NAME', '')
     name_prefix = f'{service_name}_' if service_name else ''
     url = settings['ARTIFACT_URL']
+    sub_branch = settings['BRANCH']
+    if sub_branch == '${GENDO_BRANCH}':
+        sub_branch = 'master'
+    print(f'git branch: {sub_branch}')
```

Comment:
@singetta // https://github.com/HardBoiledSmith/magi/pull/643/files#diff-ddb889719aa075b6e5d62781865343c5R929 에 들어가는 값들은 필수 값들이 대부분이어서 이런 경우는 아래처럼 optional로 처리하는게 좋을것 같습니다. 
```suggestion
    dv_branch = settings.get('BRANCH', 'master')
```